cli/internal/action/exportDocumentation.go

```diff
@@ -265,3 +271,52 @@ Documentation exported from `+"`"+`%s`+"`"+`
 
 	return
 }
+
+func exportLlmsFullTxt(documents []*docs.FilteredDoc, outputPath string) (err error) {
+	slog.Info(fmt.Sprintf("Exporting %d documents to %s", len(documents), outputPath))
+
+	// Sort documents
+	sort.SliceStable(documents, func(i int, j int) bool {
+		if documents[i].Document.SourceID < documents[j].Document.SourceID {
+			return true
+		}
+		if documents[i].Document.SourceID > documents[j].Document.SourceID {
+			return false
+		}
+		return documents[i].Document.ID < documents[j].Document.ID
+	})
+
+	// Format output
+	var str strings.Builder
+	for i, document := range documents {
+		var title string
+		if len(document.Sections) > 0 {
+			title = document.Sections[0].Section.Name
+		} else {
+			title = document.Document.ID
+		}
+		if i > 0 {
+			str.WriteString("\n\n")
```

Comment:
Based on what I can see in the golden file, I think you need one more `\n` to get the two empty lines between documents we discussed
```suggestion
			str.WriteString("\n\n\n")
```

Reply:
Yeah, you're right. When there is no trailing newline I would need 3. I'll add the third newline and trim the contents.